TestApp/DeviceAgentUnitTests/AutomationActions/Gestures/EventSynthesis/TouchPathTests.m

```diff
@@ -33,8 +33,9 @@ - (void)expectTouchPath:(TouchPath *)touchPath
 
     XCPointerEventPath *eventPath = touchPath.eventPath;
     expect(eventPath.pointerEvents.count).to.equal(1);
-    expect([eventPath.pointerEvents[0] coordinate]).to.equal(point);
-    expect([eventPath.pointerEvents[0] offset]).to.equal(seconds);
+    XCPointerEvent *event = eventPath.pointerEvents[0];
+    expect(event.coordinate).to.equal(point);
+    expect(event.offset).to.equal(seconds);
```

Comment:
We need this change because `[eventPath.pointerEvents[0] offset]` became ambiguous in Xcode 11 and it leads to the build-time failure.
By this changes we explicitly specify type of `event` and address its properties `coordinate` and `offset` directly, not via method-calling (`[object method`)) syntax.